src/tests/test_dte_data_models.py

```diff
@@ -1060,111 +1165,123 @@ def test_validate_emisor_razon_social_empty(self) -> None:
         expected_validation_errors = [
             {
                 'loc': ('emisor_razon_social',),
-                'msg': "Value must not be empty.",
+                'msg': 'Value error, Value must not be empty.',
                 'type': 'value_error',
             },
         ]
 
-        with self.assertRaises(pydantic.v1.ValidationError) as assert_raises_cm:
+        with self.assertRaises(pydantic.ValidationError) as assert_raises_cm:
             dataclasses.replace(
                 self.dte_xml_data_1,
                 emisor_razon_social='',
             )
 
-        validation_errors = assert_raises_cm.exception.errors()
+        validation_errors = assert_raises_cm.exception.errors(
+            include_context=False,
+            include_input=False,
+            include_url=False,
+        )
         self.assertEqual(len(validation_errors), len(expected_validation_errors))
-        for expected_validation_error in expected_validation_errors:
-            self.assertIn(expected_validation_error, validation_errors)
+        self.assertEqual(validation_errors, expected_validation_errors)
 
     def test_validate_receptor_razon_social_empty(self) -> None:
         expected_validation_errors = [
             {
                 'loc': ('receptor_razon_social',),
-                'msg': "Value must not be empty.",
+                'msg': 'Value error, Value must not be empty.',
                 'type': 'value_error',
             },
         ]
 
-        with self.assertRaises(pydantic.v1.ValidationError) as assert_raises_cm:
+        with self.assertRaises(pydantic.ValidationError) as assert_raises_cm:
             dataclasses.replace(
                 self.dte_xml_data_1,
                 receptor_razon_social='',
             )
-
-        validation_errors = assert_raises_cm.exception.errors()
-        self.assertEqual(len(validation_errors), len(expected_validation_errors))
-        for expected_validation_error in expected_validation_errors:
-            self.assertIn(expected_validation_error, validation_errors)
+        validation_errors = assert_raises_cm.exception.errors(
```

Comment:
Preserve empty line after `)`.

```suggestion

        validation_errors = assert_raises_cm.exception.errors(
```